runtime/laos/src/configs/democracy.rs

```diff
@@ -0,0 +1,85 @@
+use super::collective::{
+	AllOfCouncil, AllOfTechnicalCommittee, CouncilCollective, HalfOfCouncil, TechnicalCommittee,
+	TechnicalCommitteeMajority, TwoThirdOfCouncil,
+};
+use crate::{
+	currency::UNIT, weights, AccountId, Balance, Balances, BlockNumber, OriginCaller, Preimage,
+	Runtime, RuntimeEvent, Scheduler, Treasury,
+};
+use frame_support::{parameter_types, traits::EitherOfDiverse};
+use frame_system::{EnsureRoot, EnsureSigned};
+use parachains_common::{DAYS, HOURS, MINUTES};
+use polkadot_runtime_common::prod_or_fast;
+
+parameter_types! {
+	pub  LaunchPeriod: BlockNumber = prod_or_fast!(7 * DAYS, MINUTES, "LAUNCH_PERIOD");
+	pub  VotingPeriod: BlockNumber = prod_or_fast!(7 * DAYS, MINUTES, "VOTING_PERIOD");
+	pub  FastTrackVotingPeriod: BlockNumber = prod_or_fast!(3 * HOURS, MINUTES, "FAST_TRACK_VOTING_PERIOD");
+	pub  EnactmentPeriod: BlockNumber = prod_or_fast!(8 * DAYS, MINUTES, "ENACTMENT_PERIOD");
+	pub  CooloffPeriod: BlockNumber = prod_or_fast!(7 * DAYS, MINUTES, "COOLOFF_PERIOD");
+	pub const MaxProposals: u32 = 100;
+	pub const InstantAllowed: bool = false;
+	pub const MinimumDeposit: Balance = 1000 * UNIT;
+	pub const MaxVotes: u32 = 100;
+	pub const MaxDeposits: u32 = 100;
+	pub const MaxBlacklisted: u32 = 100;
+}
+
+impl pallet_democracy::Config for Runtime {
+	type BlacklistOrigin = EnsureRoot<AccountId>;
+	// To cancel a proposal before it has been passed, the technical committee must be unanimous or
+	// Root must agree.
+	type CancelProposalOrigin = EitherOfDiverse<EnsureRoot<AccountId>, AllOfTechnicalCommittee>;
+	// To cancel a proposal which has been passed, 2/3 of the council must agree to it.
+	type CancellationOrigin = TwoThirdOfCouncil;
+	/// Period in blocks where an external proposal may not be re-submitted
+	/// after being vetoed.
+	type CooloffPeriod = CooloffPeriod;
+	type Currency = Balances;
+	/// The minimum period of locking and the period between a proposal being
+	/// approved and enacted.
+	///
+	/// It should generally be a little more than the unstake period to ensure
+	/// that voting stakers have an opportunity to remove themselves from the
+	/// system in the case where they are on the losing side of a vote.
+	type EnactmentPeriod = EnactmentPeriod;
+	/// A unanimous council can have the next scheduled referendum be a straight
+	/// default-carries (NTB) vote.
+	type ExternalDefaultOrigin = AllOfCouncil;
+	/// A simple-majority can have the next scheduled referendum be a straight
+	/// majority-carries vote.
+	type ExternalMajorityOrigin =
+		pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 2>;
```

Comment:
comment above says _simple-majority_ whereas `pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 2>` is `HalfOfCouncil`